src/resources/js/v2/tickets-commerce.js

```diff
@@ -84,6 +88,37 @@ tribe.tickets.commerce = {};
 		$document.on( obj.customEvents.hideLoader, obj.loaderHide );
 	}
 
+	/**
+	 * Display the notice component.
+	 *
+	 * @since TBD
+	 *
+	 * @param {Object} data Data for notice.
+	 */
+	obj.showNotice = function( data = {} ) {
+		const $notice = $( obj.selectors.noticeItem );
+		const $content = $notice.find( obj.selectors.noticeContent );
+		const $title = $notice.find( obj.selectors.noticeTitle );
```

Comment:
There's a chance that the title or content can be missing from the markup, let's add some check in case we have that so that the JS doesn't fail.